proto/ecosystem-incentive/event.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "ecosystem-incentive/ecosystem_incentive.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+message EventRegister {
+  string incentive_unit_id = 1 [ (gogoproto.moretags) = "yaml:\"incentive_unit_id\"" ];
+  repeated SubjectInfo subject_info_list = 2 [
+    (gogoproto.moretags) = "yaml:\"subject_info_lists\"",
```

Comment:
more tag and variable name mismatch. subject_info_lists and subject_info_list

Reply:
modified it.